src/Microsoft.Win32.Registry/src/Configurations.props

```diff
@@ -2,8 +2,8 @@
 <Project ToolsVersion="14.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
   <PropertyGroup>
     <BuildConfigurations>
-      netcoreapp-Windows_NT;
-      netcoreapp-Unix;
+      netcoreapp2.0-Windows_NT;
+      netcoreapp2.0-Unix;
       netstandard;
```

Comment:
We already do that starting with @mellinoe's change https://github.com/dotnet/corefx/pull/22833/files#diff-8b54f6635c302b88758bf0c5d9e4fe02R5.  If you want that characteristic then we just need to go in and add `netcoreapp` (unversioned) configurations for these that are the same as `netcoreapp2.0` and just exclude them from the PackageConfigurations.